test/ePO-default-config/config-parameters.xsl

```diff
@@ -55,6 +55,7 @@
     <xsl:variable name="controlledListType" select="'epo:Code'"/>
     <!-- Acceptable stereotypes -->
     <xsl:variable name="stereotypeValidOnAttributes" select="()"/>
+    <xsl:variable name="stereotypeValidOnObjects" select="()"/>
```

Comment:
Beautiful variables. Very versatile and powerful script. This gives a of control